server/routes/auth.ts

```diff
@@ -0,0 +1,13 @@
+import {Router} from 'express';
+
+import {register, login} from '../controllers/auth.js';
+
+const router: Router = Router();
+
+// Register
+router.post('/register', register);
+
+// Login
```

Comment:
I think these comments are useless because they don't provide more information than we get from names of these controller methods. What is your opinion?

Reply:
yes I agree